src/librustdoc/clean/inline.rs

```diff
@@ -593,7 +594,7 @@ pub(crate) fn build_impl(
         });
     }
 
-    let (merged_attrs, cfg) = merge_attrs(cx, load_attrs(cx, did), attrs);
+    let (merged_attrs, cfg) = merge_attrs(cx, load_attrs(cx, did), attrs, &mut CfgInfo::default());
```

Comment:
A little confused about the correctness of this. Seems like inlined impls should be subject to the auto_cfg specified for the place they're inlined into, but this puts them in a fresh context...?

Reply:
I'll add a comment but it's because the computation of the cfg comes later. In here we just want to get attributes, so `CfgInfo` doesn't matter.